agent/proxycfg/testing_peering.go

```diff
@@ -51,6 +51,16 @@ func TestConfigSnapshotPeering(t testing.T) *ConfigSnapshot {
 							Service: "payments-sidecar-proxy",
```

Comment:
There are two cross-peer upstreams here and I've only adjusted one of them so we can test both scenarios at once.